sidekiq/README.md

```diff
@@ -57,8 +57,24 @@ No additional installation is needed on your server.
        mappings:
          - match: "sidekiq.sidekiq.*"
            name: "sidekiq.*"
-         - match: "sidekiq.jobs.*.perform"
+         - match: 'sidekiq\.jobs\.(.*)\.perform'
            name: "sidekiq.jobs.perform"
+           match_type: "regex"
+           tags:
+             worker: "$1"
+         - match: 'sidekiq\.jobs\.(.*)\.count'
+           name: "sidekiq.jobs.count.total"
+           match_type: "regex"
+           tags:
+             worker: "$1"
+         - match: 'sidekiq\.jobs\.(.*)\.success'
+           name: "sidekiq.jobs.success.total"
+           match_type: "regex"
+           tags:
+             worker: "$1"
+         - match: 'sidekiq\.jobs\.(.*)\.failure'
+           name: "sidekiq.jobs.failure.total"
```

Comment:
IIUC the metrics representing aggregated total are `sidekiq.jobs.count/success/failure`.

And the per job metrics are like `jobs.Foo.BarWorker.count` (what we try to match here I think). If that's correct, I think we should map it like follow

```
         - match: 'sidekiq\.jobs\.(.*)\.(count|success|failure)'
           name: "sidekiq.jobs.worker.$2"
           match_type: "regex"
           tags:
             worker: "$1"
```

To get metrics like `sidekiq.jobs.job.count`, `sidekiq.jobs.job.success`, `sidekiq.jobs.job.failure`. Using total might be confusing.

Reply:
That makes more sense! I will update it